GDJS/Runtime/pixi-renderers/layer-pixi-renderer.js

```diff
@@ -236,9 +266,90 @@ gdjs.LayerPixiRenderer.prototype.enableEffect = function(name, value) {
  * @param {string} name The effect name
  * @return {boolean} true if the filter is enabled
  */
-gdjs.LayerPixiRenderer.prototype.isEffectEnabled = function(name) {
+gdjs.LayerPixiRenderer.prototype.isEffectEnabled = function (name) {
   var filter = this._filters[name];
   if (!filter) return false;
 
   return gdjs.PixiFiltersTools.isEffectEnabled(filter);
 };
+
+/**
+ * Updates the render texture, if it exists.
+ * Also, render texture is cleared with a specified clear color.
+ */
+gdjs.LayerPixiRenderer.prototype._updateRenderTexture = function () {
+  if (!this._pixiRenderer) return;
+
+  if (!this._renderTexture) {
+    this._oldWidth = this._pixiRenderer.screen.width;
+    this._oldHeight = this._pixiRenderer.screen.height;
+
+    var width = this._oldWidth;
+    var height = this._oldHeight;
+    var resolution = this._pixiRenderer.resolution;
+    this._renderTexture = PIXI.RenderTexture.create({
+      width,
+      height,
+      resolution,
+    });
+    this._renderTexture.baseTexture.scaleMode = PIXI.SCALE_MODES.LINEAR;
+  }
+
+  if (
+    this._oldWidth !== this._pixiRenderer.screen.width ||
+    this._oldHeight !== this._pixiRenderer.screen.height
+  ) {
+    this._renderTexture.resize(
+      this._pixiRenderer.screen.width,
+      this._pixiRenderer.screen.height
+    );
+    this._oldWidth = this._pixiRenderer.screen.width;
+    this._oldHeight = this._pixiRenderer.screen.height;
+  }
+
+  var oldRenderTexture = this._pixiRenderer.renderTexture.current;
+  var oldSourceFrame = this._pixiRenderer.renderTexture.sourceFrame;
+
+  this._pixiRenderer.renderTexture.bind(this._renderTexture);
+  this._pixiRenderer.renderTexture.clear(this._clearColor);
+
+  this._pixiRenderer.render(this._pixiContainer, this._renderTexture, false);
+  this._pixiRenderer.renderTexture.bind(
+    oldRenderTexture,
+    oldSourceFrame,
+    undefined
+  );
+};
+
+/**
+ * Get the render texture used to display this layer. If render texture is not used, it is created.
+ * @returns {?PIXI.RenderTexture} RenderTexture of the container.
+ */
+gdjs.LayerPixiRenderer.prototype.getRenderTexture = function () {
```

Comment:
Is this useful? Can we make this private? The fact that a render texture exists is an implementation details, the outside world should not know about this.

Reply:
Yup, it's not being used at all right now, should be dead code then 😬. I'll remove it, for now, we can always add it later if required :)